src/jit/codegencommon.cpp

```diff
@@ -10582,6 +10595,16 @@ void CodeGen::genPreserveCalleeSavedFltRegs(unsigned lclFrameSize)
         regMaskTP regBit = genRegMask(reg);
         if ((regBit & regMask) != 0)
         {
+#ifdef FEATURE_AVX_SUPPORT
+            // Vzeroupper needs to be issued before copyIns (vmovupd etc.) that are AVX instructions
+            if (compiler->getFloatingPointInstructionSet() == InstructionSet_AVX &&
+                !bVzeroupperIssued &&
+				getEmitter()->IsAVXInstruction(copyIns))
+            {
+                instGen(INS_vzeroupper);
+                bVzeroupperIssued = true;
+            }
+#endif
```

Comment:
If we have AVX copy instruction such as vmovupd, it is 128bit AVX instruction, the verzoupper instruction needs to be issued before the AVX copy instruction (vmovupd etc.) to avoid SSE->AVX transition penalty